### DISCUSSION TITLE
use ray in composition API

### DISCUSSION THREAD

**Discussion by lrandom** [permafrost-dev/vue-ray#33] [Q&A] [answered]
does Ray support composition API ?, because I did not see it in the doc 
Best regard 
Luan

**Comment by patinthehat** [comment]
@lrandom What do you mean by "support composition API"?  

If you're asking if `vue-ray` will work when used alongside the composition API, the answer is yes.

**Comment by patinthehat** [comment, marked as answer]
@lrandom see version 2.0 of vue-ray, just released! it has full composition api support for vue 3.
